packages/iTwinUI-react/src/core/Slider/Slider.test.tsx

```diff
@@ -62,6 +63,32 @@ it('should not render thumbs if no values are defined', () => {
   expect(container.querySelector('.iui-slider-thumb')).toBeFalsy();
 });
 
+it('should not render track if value is below specified min/max', () => {
+  const { container } = render(<Slider values={[10]} min={20} max={40} />);
+  expect(
+    container.querySelector('.iui-slider-component-container'),
+  ).toBeTruthy();
+  expect(container.querySelector('.iui-slider-container')).toBeTruthy();
```

Comment:
Not sure if we need all of these `toBeTruthy()` statements. I'd remove checks for the `.iui-slider-container` and `.iui-slider-rail`

Reply:
removed